.github/workflows/build.yml

```
@@ -165,7 +165,7 @@ jobs:
         if: ${{ inputs.production && matrix.profiles.name == 'release' }}
         run: |
           mkdir -p artifact
-          tar czvf artifact/examples.tar.gz target/wasm32-unknown-unknown/release/*.wasm
+          tar czvf artifact/examples.tar.gz target/release/wasms/*.wasm
```

It would be nice to have path like this to be consistent how `cargo` creates folders for specified target (e.g. `gear-wasm` can be imagined as virtual target)

```suggestion
          tar czvf artifact/examples.tar.gz target/gear-wasm/release/*.wasm
```

---

Good idea, but I don't like naming. Maybe `wasm32-gear` would be more suitable?